src/middlewares/EventTracker.js

```diff
@@ -1,4 +1,3 @@
-/* global Piwik */
```

Comment:
This middleware has changed: instead of having a single middleware that decides if an action should be tracked and tracking it, there are now 2 separate middlewares. This one is custom for each application and only sets a flag in the action, which will be picked up by the Piwik middleware later in the chain.